services/src/tpu_utils/tpu_connection_manager.rs

```diff
@@ -249,9 +249,9 @@ impl TpuConnectionManager {
     pub async fn new(
         certificate: rustls::Certificate,
         key: rustls::PrivateKey,
-        fanout: usize,
+        _fanout: usize,
     ) -> Self {
-        let number_of_clients = fanout * 4;
+        let number_of_clients = 1; // fanout * 4;
```

Comment:
Do we make it as an argument later then ?

Reply:
yes, I want to see if it works first - and if it does, we can drop the RotatingQueue entirely